distro/src/app/components/HomePageGrid/styles/HomePageGrid.module.scss

```
@@ -1,46 +1,45 @@
+  @include breakpoint('lg') {
+    grid-template-columns: repeat(3, 10rem);
   }
 }
 
+.skeletonTile {
+  height: 8.125rem;
+}
+
 .emptyState {
   @include type-style("label-01");
   text-align: center;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)